kubernetes/config.saat.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  # Node specific vars
+  USE_CSRF_PROTECTION: "true"
+  SECURITY_REFERRER_POLICY: "origin"
+  HPKP_MAX_AGE: "2592000"
+  HPKP_SHA256S: "Set-proper-SHA256s"
+  NODE_ENV: "production"
+  UV_THREADPOOL_SIZE: "64"
+  NODE_CONFIG_DIR: "./config"
+  TS_BASE_URL: "./src/main"
+
+  # Logging vars
+  REFORM_TEAM: "ccd"
+  REFORM_SERVICE_NAME: "admin-web"
+  REFORM_ENVIRONMENT: "aat"
+
+  # Application vars
+  # IDAM
+  IDAM_BASE_URL: "https://preprod-idamapi.reform.hmcts.net:3511"
```

SAAT is pointing to IDAM test, not preprod:
```suggestion
  IDAM_BASE_URL: "http://betaDevBccidamAppLB.reform.hmcts.net"
```